internal/server/errors_test.go

```diff
@@ -80,15 +80,32 @@ func TestSendAPIError(t *testing.T) {
 			result: api.Error{Code: http.StatusNotImplemented, Message: "not implemented"},
 		},
 		{
-			err: data.UniqueConstraintError{Table: "user", Column: "name"},
+			err: fmt.Errorf("with context: %w",
+				data.UniqueConstraintError{Table: "user", Column: "name"}),
 			result: api.Error{
 				Code:    http.StatusConflict,
-				Message: "a user with that name already exists",
+				Message: "with context: a user with that name already exists",
```

Comment:
do we want the context for user-facing messages?

Reply:
I guess we probably do, since it's unlikely to be sensitive. This was the previous behaviour, but we didn't have any test that demonstrated it. I kept that behaviour for now.

It might be fine to remove it for any error that has structured field errors, since maybe those will provide enough context.